routes/api.js

```diff
@@ -1,10 +1,40 @@
 const express = require('express');
 const router = express.Router();
 
+const mail = require('./mailService');
+const authenticationHelpers = require('./authenticationHelpers');
+const models = require('../database/models');
+
+const localUser = models.user_account;
+
 /* GET api listing. */
-router.get('/getUserInfo', (req, res) => {
+router.get('/getUserInfo', authenticationHelpers.isAuthOrRedirect, (req, res) => {
     res.status(200).json({user: req.user});
 });
 
-module.exports = router;
+router.get('/username', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    const email = req.user.email;
+    localUser.findAll({
+        where:{
+            email: email
+        }}).then(data => {
+            const username = `${data[0].fname} ${data[0].lname}`;
+            res.status(200).json({username:username});
+        }).catch(error => {
+            if(error){
+                res.status(500).json({error:'Something went wrong'});
+            }
+        });
+});
 
+router.get('/mailpcpolicy', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+    const email=req.user.email;
+    mail.mailOptions.to=email;
+    mail.mailOptions.subject='Peace Corps Policy';
+    mail.mailOptions.html='<H2> Peace Corps Policy </H2>';
+    mail.smtpTransport.sendMail(mail.mailOptions, function(error){
+        error ? res.status(500).json({error:'Something Went Wrong! Try again later.'}) : res.json({message:'Mail Sent Succesfully.'});
```

Comment:
did you try to see if this works? i've always thought ternary operator requires an assignment